homework.md

```diff
@@ -4,110 +4,107 @@
 
 What is 2 + 2?
 
-5
+4
 
 ## Question 2
 
 What is the opposite of Black?
 
-Purple
+White
 
 ## Question 3
 
-What is the  2 * 3 * 6 * 1 * 10 * 34?
+What is the 2 _ 3 _ 6 _ 1 _ 10 \* 34?
 
-1322
+12240
 
-## Question 4 
+## Question 4
 
 What does HTML stand for?
 
-Hyper Tall Massive Language
+Hyper Text Markup Language
 
 ## Question 5
 
 What is CSS useful for?
 
-Making a website go faster
+CSS is the language for describing the presentation of Web pages, including colors, layout, and fonts.
 
 ## Question 6
 
 Using CSS, how would I select every `<h1>` tag on the page?
 
-```css
-#h1 {
-
+h1 {
 }
-```
 
 ## Question 7
 
 What is JavaScript?
 
-A programming language used on the internet
+JavaScript is a text-based programming language used both on the client-side and server-side
 
 ## Question 8
 
 What three problem does Git & Github solve?
 
-When people want to show off code to each other they can put it on Github
+GitHub is a Git repository hosting service, but it adds many of its own features. While Git is a command line tool, GitHub provides a Web-based graphical interface. It also provides access control and several collaboration features, such as a wikis and basic task management tools for every project
 
 ## Question 9
 
 What happens when you `fork` a repository?
 
-You delete it
+You make a editable copy of someone else's for yourself
 
-## Question 10 
+## Question 10
 
 What happens when you clone a repostory?
 
-It send it to a friend
+It downloads the repostory from web to your computer
 
 ## Question 11
 
 What is a Pull Request?
 
-When you send a file over the internet
+Proposing your changes and requesting that someone review and pull in your contribution and merge them into their branch
 
```

Comment:
The first part of this answer is perfect, but the second part is not always the case.

Sometimes you will be the person who owns the repository and you will send a pull request to other people to ask for a review. So, the other person will not be responsible for the merge.